CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+
+# Contributing to rsfbclient
+
+Currently this github repository is the main place for official discussion (via issues and pull requests)
+
+## Making and suggesting changes
+
+### Pull request or issue?
+* For small improvements or obvious bug fixes to code or documentation you can just make a pull request.
+ * For larger changes/suggestions that modify the API or bigger architectural changes, open an issue for discussion and feedback. 
+ * If you believe something should be changed in the crate's meta information (documentation, README.md, CONTRIBUTING.md, etc) you can create an issue for that. If appropriate, please provide links to existing resources that you feel may be useful.
+* Updates to out-of-date information, and small changes to existing files can be submitted directly as a PR. 
+
+### In all cases
+* Please read the section about Github Workflow. It is important that submitted PRs pass our automated tests and are correctly formatted.
+* If you are not sure about something, please check existing documentation before creating a PR or issue.  Since this crate is still fairly young, it is considerate, but not required, to also search for information that has already been discussed in old PRs and issues.
+* Note that this repository holds the source for several crates. If proposed changes are very drastic, it may be appropriate to implement it in a separate crate and/or put it behind a feature gate.
+* You may be asked to write or modify some tests after submitting an issue or PR. We will help with this where we can, if needed. See the section on testing for more information.
+
+### A template for commit messages
+(TODO)
```

Comment:
A good suggestion for this template: https://github.com/torvalds/subsurface-for-dirk/blob/a48494d2fbed58c751e9b7e8fbff88582f9b2d02/README#L88